modules/framework/galasa-parent/dev.galasa.framework/build.gradle

```diff
@@ -20,6 +21,23 @@ dependencies {
     implementation 'org.yaml:snakeyaml'
 
     testImplementation project (':dev.galasa')
+
+    testFixturesImplementation project (':dev.galasa')
+    testFixturesImplementation platform('dev.galasa:dev.galasa.platform:'+version)
+    testFixturesImplementation 'javax.validation:validation-api'
+    testFixturesImplementation 'org.assertj:assertj-core:3.16.1'
+    testFixturesImplementation 'junit:junit'
+    testFixturesImplementation 'commons-io:commons-io'
+    testFixturesImplementation 'org.apache.felix:org.apache.felix.bundlerepository'
+    testFixturesImplementation 'dev.galasa:dev.galasa.wrapping.gson'
+    testFixturesImplementation 'org.apache.bcel:bcel'
+    testFixturesImplementation 'org.apache.commons:commons-lang3'
+    testFixturesImplementation project (':dev.galasa.framework.maven.repository.spi')
+    testFixturesImplementation 'org.apache.logging.log4j:log4j-api'
+    testFixturesImplementation 'org.apache.logging.log4j:log4j-core'
+    testFixturesImplementation 'org.apache.logging.log4j:log4j-slf4j-impl'
+    testFixturesImplementation 'org.yaml:snakeyaml'
```

Comment:
Might be worth checking if we can remove any unnecessary dependencies for these test fixtures

Reply:
Yea. Some of them have gone now.